File: 김용성/최대공약수와최소공배수.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;
int gcd(int n, int m)
{
    while(m != 0)
    {
        int temp = n % m;
        n = m;
        m = temp;
    }
    
    return n;
}

int lcm(int n, int m)
{
    int g = gcd(n,m);
    return ( (n/g)  * (m / g) ) * g;
}

vector<int> solution(int n, int m) {
    vector<int> answer;
    answer.push_back(gcd(n,m));
    answer.push_back(lcm(n,m));
    cout << answer[0] <<" "<< answer[1]<< endl;
    return answer;
}

File: 조선빈/Lv.1/최대공약수와 최소공배수.py
def solution(n, m):
    answer = []
    for i in range(min(n,m),0,-1):
        if n%i == 0 and m%i == 0:
            answer.append(i)
            break
    for j in range(max(n,m),n*m+1):
        if j%n == 0 and j%m == 0:
            answer.append(j)
            break
    return answer
